### PR TITLE
Remove description duplication in ES content

### PR DESCRIPTION
**Pull request by dwhenry**
This duplication added to the complexity of the code 
as we needed to look in multiple places on a format 
specific basis. Removing this functionality simplifies
the code, if we do need to re-introduce boosting for 
this formats we should do this using the standard 
boosting functionality.

We need to check if these changes have any effect on the health check sore before we merge.

https://trello.com/c/OBhgB1Lw/531-code-tidy-up

### CODE REVIEW

File: lib/govuk_index/presenters/common_fields_presenter.rb
Comment (MatMoore) [inline lib/govuk_index/presenters/common_fields_presenter.rb:28]: I think removing this is the right thing to do, but it seems that policy pages don't have a `description` set, which means they won't display anything for the description on results pages after this change. Example data: https://www.gov.uk/api/content/government/policies/academies-and-free-schools

This is also affecting external search right now - eg googling brexit gives you

> Brexit. From: Department for Exiting the European Union. The UK voted to leave the European Union in the 23 June 2016 EU referendum. The government is now preparing to leave the EU in the best possible way for the UK's national interest. Subscribe to email alerts. Search. Document type. Case study

Instead of just 

> The UK voted to leave the European Union in the 23 June 2016 EU referendum. The government is now preparing to leave the EU in the best possible way for the UK’s national interest.

Before we merge this PR, we should make a change to policy publisher to actually populate descriptions.

**Comment by dwhenry** [comment]
@MatMoore the policy update is now being deployed so can I get a approval on this please.

**Review by MatMoore — APPROVED** [review]

**Pull request merged**
